lib/blueprinter/formatters/date_time_formatter.rb

```diff
@@ -3,12 +3,11 @@ class DateTimeFormatter
     InvalidDateTimeFormatterError = Class.new(BlueprinterError)
 
     def format(value, options)
-      return value if value.nil?
+      return value if value.nil? || !(field_format = options[:datetime_format] || Blueprinter.configuration.datetime_format)
```

Comment:
Maybe smth like this for the sake of readability?
```suggestion
      return value if value.nil?

      field_format = options[:datetime_format] || Blueprinter.configuration.datetime_format
      return value if field_format.nil?
```